app/src/main/java/org/gdg/frisbee/android/pulse/PulseFragment.java

```diff
@@ -180,9 +195,15 @@ public void onListItemClick(ListView l, View v, int position, long id) {
         if (mTarget.equals(GLOBAL)) {
             mListener.openPulse(pulse.getKey());
         } else {
-            Intent chapterIntent = new Intent(getActivity(), MainActivity.class);
-            chapterIntent.putExtra(Const.EXTRA_CHAPTER_ID, pulse.getValue().getId());
-            startActivity(chapterIntent);
+            if (v.isEnabled()) {
```

Comment:
What about just disabling the click? It is grayed out. I think it would be simpler that way. 
We will need translation for the new string.